app/decorators/mod_action_decorator.rb

```diff
@@ -67,15 +67,15 @@ def format_description
 
       ### Avoid Posting ###
     when "avoid_posting_create"
-      "Created avoid posting entry for artist \"#{vals['artist_name']}\":/artists/show_or_new?name=#{vals['artist_name']}"
+      "Created \"avoid posting entry\":/avoid_postings/#{vals['id']} for [[#{vals['artist_name']}]]"
```

Comment:
I think it would be preferable to have the link list the avoid posting entry number, to match the format of other things.
So, ex. `\"avoid posting #{vals['id']}\":/avoid_postings/#{vals['id']}`